nixos/modules/services/desktops/gnome3/at-spi2-core.nix

```diff
@@ -34,6 +34,8 @@ with lib;
 
     services.dbus.packages = [ pkgs.at_spi2_core ];
 
+    systemd.packages = [ pkgs.at_spi2_core ];
```

Comment:
Is this a user service or system service?

Reply:
It is a user service (located in `$out/lib/systemd/user/at-spi-dbus-bus.service`).

Reply:
@Mic92 Is there anything I should be aware of regarding the user – system distinction?